_posts/2022-04-25-tardigrade-launch.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title:  "Project Tardigrade Is Ready For Early Users/Contributors"
+author: Andrii Rosa, Lukasz Osipiuk, Zebing Lin, Brian Zhan
+excerpt_separator: <!--more-->
+---
+
+After 6 months of challenging work, we’ve launched Project Tardigrade. 
+It’s available right now! Read on to learn how Project Tardigrade works. 
+It required some significant and fascinating engineering to get us there. 
+I think you’ll enjoy hearing about it.
+
+ <!--more-->
+
+## What is Project Tardigrade
+
+If you’ve done big data analysis before, you probably had to guess which engine to to use based on the expected memory load. 
+If you guessed too low, you would be stuck converting your query to run elsewhere. 
+If you guessed too high, you wouldn’t be able to take advantage of Trino (like query federation, speed, standard SQL). Is there be a better way?
+
+Project Tardigrade means you don’t have to think about what memory usage. Of course there are nodes hard at work handling your query’s requests, but that’s our problem, not your’s. 
+We do all the hard work behind the scenes of allocating, configuring, and maintaining the queries. 
+Instead of spending time choosing between engines, you spend your time doing analytics and delivering business value.
+ 
+We’ve created an innovative architecture that allows us to do granular retries. 
+This means that when your long-running queries fail, they don’t have to start from scratch: they’ll restart from the latest checkpoint. 
+See the below architecture diagram. The downside is that short-running queries will slow down slightly. 
+But long-running queries will often land faster thanks to restarting from a checkpoint. 
+And most importantly, your heart won’t skip a beat when you wake up in the morning wondering whether that query landed on time: your long-running queries will have far more reliable landing times:
+<img src="/assets/blog/tardigrade-launch/tardigrade-diagram.png"/>
+
+## What Have We Done To Test It Out?
+
+Since there's no publically available testing query set for ETL use cases we had to hand craft hundreeds of ETL like queries based on the TPC-H and TPC-DS data set.
```

Comment:
```suggestion
Since there's no publically available testing query set for ETL use cases we had to hand craft hundreeds of ETL-like queries based on the TPC-H and TPC-DS data set.
```